http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/NodepoolComponent.scala

```diff
@@ -119,13 +120,14 @@ object nodepoolQuery extends TableQuery(new NodepoolTable(_)) {
     n.isDefault
   )
 
-  def claimNodepool(clusterId: KubernetesClusterLeoId)(implicit ec: ExecutionContext): DBIO[Option[Nodepool]] =
+  def claimNodepool(clusterId: KubernetesClusterLeoId,
+                    userEmail: WorkbenchEmail)(implicit ec: ExecutionContext): DBIO[Option[Nodepool]] =
     for {
       unclaimedNodepoolOpt <- findActiveByClusterIdQuery(clusterId)
         .filter(_.status === (NodepoolStatus.Unclaimed: NodepoolStatus))
         .result
         .headOption
-      _ <- unclaimedNodepoolOpt.traverse(rec => updateStatus(rec.id, NodepoolStatus.Running))
+      _ <- unclaimedNodepoolOpt.traverse(rec => updateStatusAndCreator(rec.id, NodepoolStatus.Running, userEmail))
```

Comment:
Here we must update the creator when we claim a nodepool, since we use the creator field to determine which nodepool belongs to the user